cma/evolution_strategy.py

```diff
@@ -4646,4 +4652,28 @@ def update(es):
     es.objective_function_complements = [_al]
     es.augmented_lagrangian = _al
     es.best_feasible = best_feasible_solution
+
+    if post_optimization:
+        positive_constraints = np.where(np.array(g(es.result.xfavorite)) > 0)
+        if len(positive_constraints[0]) > 0:
+            x_post_opt, es_post_opt = fmin_con(lambda x: np.sum(np.square(np.array(g(x))[positive_constraints])),
+                                               es.result.xfavorite, sigma0, g=g, h=h, **kwargs)
```

Comment:
Taking the original `sigma0` seems a pretty bad choice for post-optimization.

Reply:
We did think of this issue, from our tests (we plotted the value of evaluation of the post-optimization solution versus sigma0_post_opt), choosing sigma0_post_opt = sigma0 / 100 would be a better parameter to choose.